datalad/core/distributed/push.py

```diff
@@ -229,7 +229,6 @@ def __call__(
                     if sr
                     else 'No targets configured in dataset.'))
             return
-
```

Comment:
Noooo... my favorite empty line...why did it have to die....life is not fair...it feels so meaningless... ;-)

Reply:
I will bring it back to life ;-)